packages/core/src/test/applicationcomposer/messageHandlers/generateResourceHandler.test.ts

```diff
@@ -8,13 +8,11 @@ import sinon from 'sinon'
 import { createTemplate, createWebviewContext } from '../utils'
 import { generateResourceHandler } from '../../../applicationcomposer/messageHandlers/generateResourceHandler'
 import { Command, MessageType } from '../../../applicationcomposer/types'
-import * as env from '../../../shared/vscode/env'
+import { skipIfVscodeBelow } from '../../testUtil'
 
 describe('generateResourceHandler', function () {
     it('amazon q is not installed', async function () {
-        if (env.isMinVscode('1.89.0')) {
-            this.skip()
-        }
+        skipIfVscodeBelow(this, '1.89.0')
```

Comment:
The purpose of isMinVscode is to check if the min vscode is currently running. The purpose of the `throwWhen` arg, is to raise an error so that we know to revisit this when we bump the min vscode version.

Reply:
It feels like `isMinVscode` is doing two things, so I factored out the throwing logic into its own helper function: `throwIfMinVscodeGte`. Whether we are on the minimum is irrelevant for this use case, so taking out the throwing logic allows me to reuse the piece of the function I want.